### PR TITLE
Improve layout density of Name, Avatar, and Date, and Actions (join, etc)

### PR DESCRIPTION
**Pull request by bnvk**
I've made a bit of progress towards the goal of this above [titled issue](https://matrix.org/jira/browse/SYWEB-318) in the Jira tracker!

Still a few more tweaks to do before I'd say close it though. But I think this is already a UI improvement!

### CODE REVIEW

File: syweb/webclient/app/app.css [hunk 1]
```diff
@@ -698,11 +695,11 @@ textarea, input {
...
-    font-size: 12px;
-/*    
-    margin-top: 5px;
-    margin-bottom: -9px;
-*/
+    padding-left: 1em;
+	color: #a9a9a9;
+    font-size: 13px;
+	font-weight: bold;
+	font-family: Helvetica, Arial, sans-serif;
```
Comment (jancborchardt) [inline syweb/webclient/app/app.css]: Tabs / number of spaces ;)

**Comment by jancborchardt** [comment]
From looking at your screen it was pretty neat. Will test thoroughly tomorrow! :)

**Comment by Kegsay** [comment]
In the future, please base your PR off ``develop``, thanks!

**Comment by jancborchardt** [comment]
Nice that this is in now! :)

**Comment by bnvk** [comment]
@Kegsay ack on that. Was a newcomer mistake. Already working in `develop` branch now ;)

**Pull request merged**